tests/test_vtp.rb

```diff
@@ -176,6 +174,9 @@ def test_password_valid
     end
   end
 
+  # this fails on n7k running helsinki image but it
```

Comment:
Please add the use version number `7.3(0)D1(1) ` instead of the internal name `helsinki`